# Приводим строку к целому

# Нет ошибки
a = int("234")

# ValueError
b = float("two")

print(b)
